virttest/qemu_storage.py

```diff
@@ -728,15 +731,15 @@ def create(self, params, ignore_errors=False):
                                 "Other errors may ensue")
                 os.makedirs(image_dirname)
 
+        if self.encryption_config.key_secret:
+            self.encryption_config.key_secret.save_to_file()
```

Comment:
Would you please elaborate on this change?

Reply:
There are two ways to create luks image.
1. \# qemu-img create --object secret,id=image1_encrypt0,data=redhat -f luks -o key-secret=image1_encrypt0 /home/test.luks 1G
2. \# qemu-img create --object secret,id=image1_encrypt0,file=/home/test.secret -f luks -o key-secret=image1_encrypt0 /home/test.luks 1G

If we use the second way to create luks image, the test.secret must be saved before image creation. Currently, saving key_secret to file after image creation.

Reply:
oh, I see, thanks for your elaboration!